<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regist</class>
 <widget class="QWidget" name="regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passRegist">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailRegist">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userRegist">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
